src/app/src/utils.js

```diff
@@ -98,18 +98,7 @@ export function downloadData(data, filename = null) {
 }
 
 export function getBoundaryShapeFilename(boundary) {
-    const now = new Date();
-
-    const year = now.getFullYear();
-    const month = now.getMonth().toString().padStart(2, '0');
-    const date = now.getDate().toString().padStart(2, '0');
-
-    return `${[
-        year,
-        month,
-        date,
-        boundary.utility.name.replaceAll(/\s+/g, '_'),
-    ].join('_')}.geojson`;
+    return `${boundary.official_name}.geojson`;
```

Comment:
With these changes, I think the usefulness of this utility function is questionable.

Reply:
Good point, will inline.

Reply:
[c5d1cfd](https://github.com/azavea/iow-boundary-tool/pull/202/commits/c5d1cfdd2c23949c4ba54422384c4e3f9e313de8)